importer/templates/importer.html

```
@@ -103,12 +109,30 @@
             </form>
         </nav>
     </div>
 </div>
+
+<!-- Import confirmation modal -->
+<div class="modal" id="import-confirm-modal">
+    <div class="modal-background" onclick="closeImportConfirmation()"></div>
+    <div class="modal-card">
+        <header class="modal-card-head">
+            <p class="modal-card-title">Confirm import</p>
+            <button class="delete" aria-label="close" onclick="closeImportConfirmation()"></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)